### PR TITLE
changes for create account modal

### PR DESCRIPTION
**Pull request by KarishmaBothara**
Made changes for the 'create account' modal.. Lillian wanted me to align the bullet points to the boxes above it and remove the help '?'..
Reviewed by Lilian
Screen Shot 2020-07-07 at 10 37 18 AM

### CODE REVIEW

File: packages/app-accounts/src/modals/Create.tsx [hunk 1]
```diff
@@ -229,26 +229,53 @@ function Create ({ className, onClose, onStatusChange, seed: propsSeed, type: pr
...
             autoFocus
             className='full'
-            help={t('Name given to this account. You can edit it. To use the account to validate or nominate, it is a good practice to append the function of the account in the name, e.g "name_you_want - stash".')}
             isError={!isNameValid}
-            label={t('name')}
+            label={t('Name your account (you can edit later)')}
             onChange={_onChangeName}
             onEnter={_onCommit}
-            placeholder={t('new account')}
+            placeholder={t('e.g. validate, nominate')}
```
Comment (holygits) [inline packages/app-accounts/src/modals/Create.tsx:236]: I'm not sure these are very good example names 🤷‍♂️

File: packages/app-accounts/src/modals/Create.tsx [hunk 2]
```diff
@@ -323,4 +337,11 @@ export default styled(Create)`
   .accounts--Creator-advanced {
     margin-top: 1rem;
   }
+  .bullet-points {
+    margin-left: 0.3rem;
+    color: rgba(78,78,78,.66);
```
Comment (holygits) [inline packages/app-accounts/src/modals/Create.tsx:342]: can put colors in the styled-themeing file as constants and re-use them here

File: packages/app-accounts/src/modals/Create.tsx (hunk 1 above)
Comment (KarishmaBothara) [reply packages/app-accounts/src/modals/Create.tsx:236]: this was as per design specification 😳

File: packages/app-accounts/src/modals/Create.tsx (hunk 2 above)
Comment (KarishmaBothara) [reply packages/app-accounts/src/modals/Create.tsx]: done

**Pull request merged**